draft-ietf-cose-bls-key-representations.md

```diff
@@ -200,6 +200,48 @@ E'(GF(p^8))
 
 {backmatter}
 
+# Appendix
+
+The following example JWK is for the Bls12381 curve where the public key is in the G1 subgroup.
+
+```
+{
+  "kty": "OKP",
+  "crv": "Bls12381G1",
+  "d": "Mt_OyD9IAsYvobHJ9NCipm6-G7zAu28FCc-saRnXhjQ",
```

Comment:
It might be worth a comment saying that these - on purpose - represent private keys.  Otherwise people might question the inclusion of "d".

Reply:
Thanks! Added a note in the beginning of the Section.